ilc/server/errorHandler/index.spec.js

```diff
@@ -0,0 +1,126 @@
+const chai = require('chai');
```

Comment:
I don't think that we need unit tests for this... Integration tests should be used instead. Now test is too fragile

Reply:
Let's discuss this...

Reply:
I wrote integration tests
Need to re-review